packages/iTwinUI-react/src/core/TimePicker/TimePicker.tsx

```diff
@@ -461,7 +650,9 @@ const TimePickerColumn = <T,>(props: TimePickerColumnProps<T>): JSX.Element => {
                 onSelectChange(value);
               }}
             >
-              {valueRenderer(value)}
+              {useCombinedRenderer && precision && valueRenderer.length === 2
+                ? valueRenderer(value, precision)
+                : valueRenderer(value)}
```

Comment:
Could we have a use example of `valueRenderer`, maybe in Storybook? I was (and other users might also be) curious to see how it works and I also wanted to test why precision is also a parameter

Reply:
`valueRenderer` is just `hourRenderer`, `minuteRenderer`, `secondRenderer`, `timeRenderer`, or `meridiemRenderer`. `valueRenderer` is the generic version of it for `TimePickerColumn`.

We don't show `TimePickerColumn`, but I can add an example for some custom renderers.

Reply:
Ohh, I see, thanks. I think I understand now what `valueRenderer` after seeing its uses on lines 470, 485, 495, etc.

But to confirm, can we not just write `valueRenderer(value, precision)`? Even if on [L485](https://github.com/iTwin/iTwinUI-react/pull/844/files/03bff18ad6a2a47d22454ee86a441c5cfea43341#diff-47435636e77eb38c7aebe52aca573c1aed481fb9e21916cc70c423b6745b88fbR485) for example we pass `hourRenderer` that just has the date param, I can we can still call `valueRenderer` with the `precision` param.

Reply:
Oh, that might be fine. I just didn't want there to be bugs if it's not okay.

Reply:
I'll try it and run the tests.

Reply:
I simplified to just `valueRenderer(value, precision)`. I also added a new story for custom renderers.

Reply:
The new story is very helpful!